applications/asset_tracker_v2/src/cloud/lwm2m_integration/lwm2m_integration.c

```diff
@@ -141,7 +141,6 @@ static void rd_client_event(struct lwm2m_ctx *client, enum lwm2m_rd_client_event
 		LOG_WRN("LWM2M_RD_CLIENT_EVENT_REGISTRATION_FAILURE");
```

Comment:
I Think it is good for set state connecting here also.

Reply:
The engine will be restarted if `LWM2M_RD_CLIENT_EVENT_REGISTRATION_FAILURE` is received.

Reply:
Note line 196